queries/java/highlights.scm

```diff
@@ -150,25 +168,30 @@
 "case"
 ] @conditional
 
-; 
+;
 
 [
 "for"
 "while"
 ] @repeat
 
-;
+; Includes
 
 "import" @include
 
 ; Punctuation
```

Comment:
We could also add "package" to includes.

Reply:
Don't want to block this PR. Just ideas for more highlights...

Reply:
> Don't want to block this PR. Just ideas for more highlights...

yeah no worry

Reply:
done